AppHandling/Run-AlPipeline.ps1

```diff
@@ -1635,6 +1654,9 @@ Write-Host -ForegroundColor Yellow @'
         "appProjectFolder" = $folder
         "appOutputFolder" = $appOutputFolder
         "appSymbolsFolder" = $appPackagesFolder
+        "isApp" = $app
+        "isTestApp" = $testApp

```

Comment:
Is that a parameter on Compile-AppInBcContainer ?
Won't this fail when calling that?
Maybe create another hashtable for event specific parameters like this.
and then add the appFile after compilation - I assume in the post compile step you want the appFile right?

Reply:
`$app` is otherwise a global variable for (see [this reference](https://github.com/microsoft/ALAppExtensions/blob/25e371e96f2651dabd2af08df11f8d3d5465d510/Build/Scripts/CompileAppInBcContainer.ps1#LL11C1-L12C9) for example).

My idea of adding the values of `$app`, `$testApp` and `$bcptTestApp` as parameters was to avoid using variables that were defined in `Run-AlPipeline.ps1` in the override scripts.

But you are right, it is breaking to simply add extra parameters.
I will give it an extra thought.